backend/src/plugins/ModActions/functions/actualMuteUserCmd.ts

```diff
@@ -42,7 +42,11 @@ export async function actualMuteUserCmd(
   }
 
   const timeUntilUnmute = args.time && humanizeDuration(args.time);
-  const reason = args.reason ? formatReasonWithAttachments(args.reason, [...msg.attachments.values()]) : undefined;
+
+  const config = pluginData.config.get();
+  const reason = args.reason
+    ? formatReasonWithAttachments(args.reason, [...msg.attachments.values()])
+    : config.default_reasons?.mute || "No reason specified";
```

Comment:
This should limit the reason to 512 characters. But now that I look at the `formatReasonWithAttachments()` function, it also does not seem to be limited. Any chance you can modify that too?